synfig-studio/src/gui/resources/css/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+set(INSTALL_DESTINATION "share/synfig/css")
+set(CSS_FILES
+	synfig.css
+	synfig.mac.css
+)
+
+# Prefix all CSS_FILES list with source directory for
+#
+# add_custom_command(COMMAND ...)
+#
+set(CSS_FILES_INPUT)
+foreach(CSS_FILE ${CSS_FILES})
+	list(APPEND CSS_FILES_INPUT ${CMAKE_CURRENT_SOURCE_DIR}/${CSS_FILE})
+endforeach()
+
+# Prefix all CSS_FILES list with destination directory for
+#
+# add_custom_command(OUTPUT ...)
+#
+set(CSS_FILES_OUTPUT)
+foreach(CSS_FILE ${CSS_FILES})
+	list(APPEND CSS_FILES_OUTPUT ${SYNFIG_BUILD_ROOT}/${INSTALL_DESTINATION}/${CSS_FILE})
+endforeach()
+
+# file(COPY..) has limitations
```

Comment:
You can use `configure_file` here with COPYONLY (https://cmake.org/cmake/help/latest/command/configure_file.html)

Reply:
Doesn't work :(

Tried this
```cmake
configure_file(${CSS_FILES} ${SYNFIG_BUILD_ROOT}/${INSTALL_DESTINATION}/${CSS_FILES} COPYONLY)
```
and this...
```cmake
configure_file(${CSS_FILES_INPUT} ${CSS_FILES_OUTPUT} COPYONLY)
```

*No such file or directory* exception was caught when building and running

Reply:
This example works for me:
```
set(INSTALL_DESTINATION "share/synfig/css")
set(CSS_FILES
    synfig.css
    synfig.mac.css
)

foreach(CSS_FILE ${CSS_FILES})
    configure_file(${CSS_FILE} ${INSTALL_DESTINATION}/${CSS_FILE} COPYONLY)
endforeach()
```

Reply:
Thanks! The earlier didn't work because I tried to copy a list. Looping each item is much better than my lengthy custom target style :P